pages/glossary/valid-language-tag.md

```diff
@@ -5,12 +5,15 @@ unambiguous: true
 objective: true
 ---
 
-A language tag consists of a [primary language subtag][] from the [language subtag registry][], optionally followed by a hyphen (`-`) and any sequence of characters.
+A [language tag][] is _valid_ if its [primary language subtag][] exists in the [language subtag registry][] with a `Type` of `language`.
```

Comment:
Just wondering if it's worth adding a link to the [Type Field section](https://tools.ietf.org/html/bcp47#section-3.1.3).